search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -135,40 +126,76 @@ class TemporalEntityHandler(
         @PathVariable entityId: String,
         @RequestParam params: MultiValueMap<String, String>
     ): ResponseEntity<*> {
-        val sub = getSubFromSecurityContext()
-        val contextLink = getContextFromLinkHeaderOrDefault(httpHeaders)
-        val mediaType = getApplicableMediaType(httpHeaders)
-
-        val withTemporalValues =
-            hasValueInOptionsParam(
-                Optional.ofNullable(params.getFirst(QUERY_PARAM_OPTIONS)),
-                OptionsParamValue.TEMPORAL_VALUES
+        return either<APIException, ResponseEntity<*>> {
```

Comment:
a bit strange to have refactored this endpoint and not the others, why is that?

typically it is not recommended to mix a feature and a refactoring in the same PR (even though it may happen you have no choice)

Reply:
Ok I take it off